tools/convert.rb

```diff
@@ -1394,6 +1394,7 @@ def updateDbItem(data)
   ItemAuthor.where(item_id: itemID).delete
   ItemContrib.where(item_id: itemID).delete
   UnitItem.where(item_id: itemID).delete
+  data['dbSection'].nil? and Item.where(id: itemID).update(section: nil)
```

Comment:
I think nulling out the section here is redundant, because `data[:dbItem].create_or_update()` saves the whole item record.

That said, this won't hurt anybody, so I'll let it go through. You might want to try without it and if that works, drop this line.

Reply:
Hmm, I did try without and section ID remained in item. I'm not sure what create_or_update does.